packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1244,3 +1244,91 @@ public class Shell {
         VM-Write-Log-Exception $_
     }
 }
+
+
+# Usage example:
+# VM-Remove-DesktopFiles -excludeFolders "PS_Transcripts", ${Env:TOOL_LIST_DIR}, "fakenet_logs" -excludeFiles "example.txt", "important.doc"
+# The function is run against both the Current User and 'Public' desktops due to some cases where desktop icons showing on
+# Current user Desktop that are only located in Public/Desktop.
+function VM-Remove-DesktopFiles {
+    param (
+        [Parameter(Mandatory=$false)]
+        [string[]]$excludeFolders,
+        [Parameter(Mandatory=$false)]
+        [string[]]$excludeFiles
+    )
+    # Ensure that "PS_Transcripts", "Desktop\Tools", and "fakenet_logs" folders are not to be deleted.
```

Comment:
[nitpick] The tools dirs is not always in `Desktop\Tools`. Although maybe it is clear what is meant, I purpose:
```suggestion
    # Ensure that the "PS_Transcripts" and "fakenet_logs" folders as well as the tools directory are not to be deleted.
```

Reply:
I agree, it isn't always on the desktop. However, because this function only removes folders that -are- on the desktop, if the Tools directory isn't located there, it will not be touched. :)

I can reword the comment to be a little more clear about that if need be.